src/extension/src/signalr-api-module/syncCommand.ts

```diff
@@ -15,7 +14,6 @@ export class SyncCommand extends CoreTemplateStudioApiCommand {
         this.commandPayload.payload!.path
       )
       .catch((error: Error) => {
-        Logger.appendLog("EXTENSION", "error", error.message);
```

Comment:
why did we remove this log?

Reply:
I removed it because it originally caused my test to error out since it uses the module vscode but my test runs independently from the extension and it was also inconsistent with sync command

I moved the logger to the catch block of attemptsync in launchexperience so it should achieve the same purpose as including it in syncCommand